Lib/test/_test_multiprocessing.py

```diff
@@ -3880,6 +3880,10 @@ class OptionalAttachSharedMemory(shared_memory.SharedMemory):
 
         sms.close()
 
+        # Test creating a shared memory segment with size greater than shared_memory._MAX_SHM_SIZE_LIMIT
+        with self.assertRaises(ValueError):
+            sms_invalid = shared_memory.SharedMemory(create=True, size=shared_memory._MAX_SHM_SIZE_LIMIT + 1)
```

Comment:
There's no need for a variable here.

Reply:
Totally agree, I will remove that.
But, if you see there are similar tests around this test assigning an unnecessary variable, should I remove those too ?